lib/puppet/type/dsc_xcomputer.rb

```diff
@@ -48,9 +48,23 @@
   newparam(:name, :namevar => true ) do
   end
 
+  # ensurable do
+  #   newvalue(:exists?) { provider.exists? }
+  #   newvalue(:present) { provider.create }
+  #   defaultto { :present }
+  # end
+
+  attr_reader :last_dsc_result
+  autonotify(:reboot) { ['dsc_reboot'] }
```

Comment:
So we still want to keep the autonotify.. but... I think we can remove the rest.

Do you want to start poking around at the code to retrieve the resource named `['dsc_reboot']` in the event that we get a `RebootRequired` back from `Invoke-DscResource`?